src/Select/Select.tsx

```diff
@@ -15,6 +15,7 @@ export interface SelectValueType {
 }
 
 export interface Props {
+  labelText?: string;
```

Comment:
Curious about the name `labelText` vs. `ariaLabel`. If it's called `ariaLabel` it feels very clear from the outside that the value passed along will eventually be used for a11y purposes (it'll eventually be used as an `aria-label` in the DOM). `labelText` on the other hand, is difficult to distinguish from `label` as far as its eventual purpose. Somebody may need to dive into the source code to figure out what it's used for and how the two are different.

This piece matters less, but I'm curious what you think of listing this new prop last on these lists rather than first. The new prop is optional and currently is not widely used, so maybe that first position in the list could be saved for the two required props (`id` and `name`)?

Reply:
good point! I'll make those edits